### ISSUE TITLE
HTTP query string 404 error

### ISSUE THREAD

**Issue by petk** [colshrapnel/phpdelusions.net#8]
Hello. Really awesome and respectable site https://phpdelusions.net We share it on Facebook in groups many times.

Would it be possible to fix one issue. Facebook adds an HTTP query string ?fbclid at the end and site 404 errors then 🙏

https://phpdelusions.net/?fbclid=123

Thank you so much.

**Comment by colshrapnel** [comment]
Man,
First of all thank you for the report and then for the insight.
It was strtok(). Somehow I am addicted to this function (as it used to be the only way to create nice one-liners), but only today I learned that it ignores "token" if it's on the first or the last position. Hence, say, `strtok("mysqli?fbclid=123", "?")` returns "mysqli", but `strtok("?fbclid=123", "?")` returns ?fbclid=123.
Changed it to `explode("?", $uri)[0];` and now it works a treat!
It explains why only the main page was affected.

Thanks again!
